### PR TITLE
Survey form

### LINKED ISSUE #145: Survery form
survey
Can I add this survey form?
Also please tell me if only html, css, js is allowed in this project? Or we can also use nodejs in our projects?

### PR DESCRIPTION
**Pull request by RiyaGupta89**
Added survey form in the repository. 
# Resolves #145 
survey

### CODE REVIEW

File: SURVEY FORM/app.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    res.sendFile(__dirname + "/index.html");
+});
+
+app.post("/", (req, res) => {
+    res.send("<h1> Thankyou for filling up this feedback form. Go back to the previous page  =) </h1>");
+})
+
+app.listen(3000, function() {
+    console.log("Server started running on port 3000");
+})
```
Comment (chicken-biryani) [inline SURVEY FORM/app.js:17]: Add a new line at the end of file

File: SURVEY FORM/public/css/style.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    opacity: 1;
+}
+
+h1{
+    color: aliceblue;
+    text-align: center;
+    margin-top: 50px;
+    margin-bottom: 0;
+    font-size: 56px;
+}
```
Comment (chicken-biryani) [inline SURVEY FORM/public/css/style.css:50]: Add a new line at the end of file

File: SURVEY FORM/index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+            <label><b>What do you like most in our Pizzas?</b></label>
+            <textarea name="" id="" cols="30" rows="5"></textarea>
+
+            <button type="submit" class="btn">Submit</button>
+            </div>
+        </form>
+
+</body>
+</html>
```
Comment (chicken-biryani) [inline SURVEY FORM/index.html:39]: Add a new line at the end of file.

File: CLASSROOM SCHEDULER WEBSITE/index.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      <hr class="margin-top hr" />
+      <h1 class="class">Your Classes</h1>
+
+      <hr class="hr"/>
+      <div class="showNotes" id="notes"></div>
+    </div>
+
+    <script src="script.js"></script>
+  </body>
+</html>
```
Comment (chicken-biryani) [inline CLASSROOM SCHEDULER WEBSITE/index.html:54]: Add a new line at the end of file.

**Review by harshita2216 — APPROVED** [review]

**Pull request merged**